docs/src/pages/components/tables/EnhancedTable.js

```diff
@@ -64,7 +64,7 @@ function getSorting(order, orderBy) {
   return order === 'desc' ? (a, b) => desc(a, b, orderBy) : (a, b) => -desc(a, b, orderBy);
 }
 
-const headRows = [
+const headItems = [
```

Comment:
What do you think of? 

```suggestion
const headCells = [
```

Reply:
That’s much better! I like it! I will change the variable in the map function as well.

Reply:
Cool :), don't forget to start with the TypeScript version and to run `yarn docs:typescript:formatted` to generate the JavaScript version.

Reply:
I was gonna ask you about the failure in testing. The test did not pass because the TypeScript version was not included in the pull request. That makes sense. 
Thanks a lot!